README.md

```diff
@@ -1508,18 +1508,18 @@ Full Stack Developer & Site Reliability Engineer based in New Zealand, intereste
 <img align="left" width="100" height="100" src="assets/images/members/kevyn.png"/>
 
 [Kevyn Bruyere](https://github.com/kevynb)
-<a href="https://www.linkedin.com/in/kevynbruyere/"><img src="assets/images/linkedin.png" width="16" height="16"></img></a>
+<a href="https://www.linkedin.com/in/kevynbruyere/"><img src="assets/images/linkedin.svg" width="16" height="16"></img></a>
 
 Independent full-stack developer with a taste for Ops and automation.
 
 <br/>
 
-<a id="josh-hemphill" href="https://github.com/josh-hemphill" target="_blank"><img src="assets/images/members/jhemphill.jpg" align="left" width="100" height="100" alt="Josh Hemphill" loading="lazy"/></a>
+<a id="josh-hemphill" href="https://github.com/josh-hemphill" target="_blank"><img src="assets/images/members/josh-hemphill.png" align="left" width="100" height="100" alt="Josh Hemphill" loading="lazy"/></a>
 
 [Josh Hemphill](https://github.com/josh-hemphill)
-<a href="https://twitter.com/spooklogical"><img src="assets/images/twitter-s.png" width="16" height="16"></img></a>
-<a href="https://www.linkedin.com/in/joshuahemphill/"><img src="assets/images/linkedin.png" width="16" height="16"></img></a>
-<a href="https://joshuahemphill.com"><img src="assets/images/www.png" width="16" height="16"></img></a>
+<a href="https://twitter.com/spooklogical"><img src="assets/images/twitter.svg" width="16" height="16"></img></a>
+<a href="https://www.linkedin.com/in/joshuahemphill/"><img src="assets/images/linkedin.svg" width="16" height="16"></img></a>
+<a href="https://joshuahemphill.com"><img src="assets/images/web.svg" width="16" height="16"></img></a>
 
 Full Stack Software Engineer / Developer specializing in Security, DevOps/DevSecOps, and ERP Integrations.
 
```

Comment:
It looks like you forgot to update my row 😉

Reply:
@josh-hemphill I did not disable it intentionally